### DISCUSSION TITLE
Gym interface

### DISCUSSION THREAD

**Discussion by onnoeberhard** [google/brax#336] [Q&A] [answered]
In `brax/envs/ant.py` in the Ant docstring, it is hinted that the environment can be created using the gym(nasium) interface with `Ant-v5`: https://github.com/google/brax/blob/main/brax/envs/ant.py#L157.
Is this really possible? I have not had any success.

Thanks!

**Comment by btaba** [comment, marked as answer]
Hi @onnoeberhard theses docstrings were copied over from brax v1. There was a colab at some point to get brax envs integrated into gynasium (https://github.com/Farama-Foundation/Gymnasium/issues/12), but it never went through
